### PR TITLE
Lower4upper acro

### PR DESCRIPTION
**Pull request by thorunna**
Handling of acronyms added to `errtokenizer.py`. It includes a list of common acronyms which is taken into account when detecting errors. Maybe this list should rather be in a config file? 
The current method can only detect and correct acronyms when the first letter is uppercase and all others lowercase.
An error class for acronyms, Z006, is also added to capitalization errors along with testing for this error class.

### CODE REVIEW

File: src/reynir_correct/errtokenizer.py [hunk 1]
```diff
@@ -104,6 +104,27 @@
     "Desember",
 )
 
+ACRONYMS = (    # HÍ og HA ganga kannski ekki hér
```
Comment (vthorsteinsson) [inline src/reynir_correct/errtokenizer.py:107]: Replace `(` by `frozenset((`

File: src/reynir_correct/errtokenizer.py [hunk 2]
```diff
@@ -104,6 +104,27 @@
...
+    "Ms"
+    "Hr",
+    "Ísí",
+    "Ksí",
+    "Kr",
+    "Fh",
+    "Ía",
+    "Ka",
+    "Hk"
+)
```
Comment (vthorsteinsson) [inline src/reynir_correct/errtokenizer.py:126]: Replace `)` by `))`

File: src/reynir_correct/errtokenizer.py [hunk 3]
```diff
@@ -104,6 +104,27 @@
...
+    "Rúv",
+    "Byko",
+    "Íbv",
+    "Pga",
+    "Em",
+    "Ví",
+    "Mr",
+    "Mh",
+    "Ms"
+    "Hr",
```
Comment (vthorsteinsson) [inline src/reynir_correct/errtokenizer.py:118]: What about "Hr. Hnetusmjör hélt tónleika í Kópavogi í kvöld"?

File: src/reynir_correct/errtokenizer.py [hunk 4]
```diff
@@ -104,6 +104,27 @@
...
 
+ACRONYMS = (    # HÍ og HA ganga kannski ekki hér
+    "Dv",
+    "Rúv",
+    "Byko",
+    "Íbv",
+    "Pga",
+    "Em",
+    "Ví",
+    "Mr",
```
Comment (vthorsteinsson) [inline src/reynir_correct/errtokenizer.py:115]: What about "Sigurður lék hlutverk Mr. Smith í leikritinu?"

File: src/reynir_correct/errtokenizer.py [hunk 5]
```diff
@@ -104,6 +104,27 @@
...
+    "Mh",
+    "Ms"
```
Comment (vthorsteinsson) [inline src/reynir_correct/errtokenizer.py:117]: What about "Guðrún lék hlutverk Ms. Abercrombie í þáttunum?"

File: src/reynir_correct/errtokenizer.py (hunk 1 above)
Comment (thorunna) [reply src/reynir_correct/errtokenizer.py]: Thanks! Have changed this

File: src/reynir_correct/errtokenizer.py (hunk 3 above)
Comment (thorunna) [reply src/reynir_correct/errtokenizer.py:118]: The period prevents 'Hr.' from being corrected and the same goes for the two other sentences, 'Mr.' and 'Ms.'. I have added testing for these sentences to show how they are handled.

**Comment by vthorsteinsson** [comment]
Looks good to me! OK to merge?

**Comment by thorunna** [comment]
It's OK for my sake!

**Comment by vthorsteinsson** [comment]
Good work, thanks!

**Pull request merged**